File: backend/src/main/java/com/ftn/activityapp/dto/planner/PlannerTaskResponseDto.java
package com.ftn.activityapp.dto.planner;

import com.ftn.activityapp.enums.PlannerTaskType;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;
import java.time.LocalTime;

@Getter
@Setter
@Builder
public class PlannerTaskResponseDto {
    private Long id;
    private String title;
    private String description;
    private LocalDate date;
    private LocalTime time;
    private PlannerTaskType taskType;
    private Boolean completed;
    private String source;
    private Long userId;
}

File: mobile-app/app/src/main/java/com/example/activityapp/data/remote/dto/planner/PlannerTaskResponseDto.kt
package com.example.activityapp.data.remote.dto.planner

data class PlannerTaskResponseDto(
    val id: Long,
    val title: String,
    val description: String?,
    val date: String,
    val time: String,
    val taskType: String,
    val completed: Boolean,
    val source: String,
    val userId: Long
)